### PR TITLE
Add GitHub Actions

### PR DESCRIPTION
**Pull request by svkirillov**

### CODE REVIEW

**Comment by xDmitriev** [comment]
I would suggest using the same way as we implemented it in sidecar and ingress repos. In these projects, we use our self-hosted runners in conjunction with Vault. Such approach allows us to manage Dockerhub creds in one place. Please, check the example below.

https://github.com/wallarm/sidecar/blob/8e0f703f7290e7770915e7fd4e7e8f27354f5558/.github/workflows/build.yaml#L16
https://github.com/wallarm/sidecar/blob/8e0f703f7290e7770915e7fd4e7e8f27354f5558/.github/workflows/build.yaml#L21

File: .github/workflows/dockerhub-push.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Build and push
+        uses: docker/build-push-action@v3
+        with:
+          context: .
+          platforms: linux/amd64,linux/arm64
+          push: true
+          tags: ${{ secrets.DOCKERHUB_USERNAME }}/gotestwaf:latest
```
Comment (xDmitriev) [inline .github/workflows/dockerhub-push.yml:30]: I'm not sure that `${{ secrets.DOCKERHUB_USERNAME }}` should be placed here, probably just `wallarm`

File: .github/workflows/dockerhub-description.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Update DockerHub Description
+        uses: peter-evans/dockerhub-description@v3
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+          repository: ${{ secrets.DOCKERHUB_USERNAME }}/gotestwaf
```
Comment (xDmitriev) [inline .github/workflows/dockerhub-description.yml:23]: Same as below. Probably just `repository: wallarm/gotestwaf`

File: .github/workflows/dockerhub-push.yml (hunk 1 above)
Comment (svkirillov) [reply .github/workflows/dockerhub-push.yml:30]: Yes, in case of `vault-action` we need to change it to `wallarm`.

Comment (svkirillov) [reply .github/workflows/dockerhub-push.yml]: Fixed - https://github.com/wallarm/gotestwaf/pull/185/commits/3d9a4babbaa744126d14d9e9af648b513afa84dc

File: .github/workflows/dockerhub-description.yml (hunk 2 above)
Comment (svkirillov) [reply .github/workflows/dockerhub-description.yml]: Fixed - https://github.com/wallarm/gotestwaf/pull/185/commits/3d9a4babbaa744126d14d9e9af648b513afa84dc

**Comment by xDmitriev** [comment]
LGTM

**Pull request merged**